ShopifySharp/Entities/SalesChannel/CreatePayment.cs

```diff
@@ -0,0 +1,34 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+using Newtonsoft.Json;
+
+namespace ShopifySharp.Entities.SalesChannel
+{
+    public partial class CreatePayment
+    {
+        [JsonProperty("request_details")]
+        public RequestDetails RequestDetails { get; set; }
+
+        [JsonProperty("amount")]
+        public string Amount { get; set; }
+
+        [JsonProperty("session_id")]
+        public string SessionId { get; set; }
+
+        [JsonProperty("unique_token")]
+        public string UniqueToken { get; set; }
+    }
+
+    public partial class RequestDetails
```

Comment:
This should be moved into its own file and renamed to something like `SalesChannelPaymentRequestDetails`.